override.conf

```diff
@@ -1,2 +1,7 @@
 [Service]
-ExtensionDirectories=/var/lib/extensions/tailscale
+ExecStartPre=
```

Comment:
(minor) Is this empty declaration (and similar ones in lines 4 and 6) necessary? The config seems to work well without them anyway, but can we remove them if they're not necessary?

Reply:
From [Freedesktop.org - systemd.unit](https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html):

> Note that for drop-in files, if one wants to remove entries from a setting that is parsed as a list (and is not a dependency), such as AssertPathExists= (or e.g. ExecStart= in service units), one needs to first clear the list before re-adding all entries except the one that is to be removed.

I think this is necessary for `ExecStart` but I really am not sure if the rest are interpreted as lists as well...